### PR TITLE
add ci check for terraform files

### PR DESCRIPTION
**Pull request by dkijania**
Explain your changes:
Since we want to remove integration test from Pull request we lost a test of our infra code. This pr introduce isolated test for our helm/terraform tests. Small adjsutments for ci-net testnet was necessary as file is out of date

Explain how you tested your changes:
Checked CI and run locally one lucy test using my terraform files

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: buildkite/src/Jobs/Test/TerraformNetworkTest.dhall [hunk 1]
```diff
@@ -0,0 +1,57 @@
+let Command = ../../Command/Base.dhall
+let RunInToolchain = ../../Command/RunInToolchain.dhall
+let Docker = ../../Command/Docker/Type.dhall
+let Size = ../../Command/Size.dhall
+
+
+let Prelude = ../../External/Prelude.dhall
```
Comment (SanabriaRusso) [inline buildkite/src/Jobs/Test/TerraformNetworkTest.dhall:7]: Are the lines above this point needed?

Comment (dkijania) [reply buildkite/src/Jobs/Test/TerraformNetworkTest.dhall:7]: Fixed

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**